src/bindings/c/CMakeLists.txt

```diff
@@ -32,6 +33,7 @@ set(C_C_LIB_SRCS
     growtree.c
     indices.c
     imports.c
+    precision.c
```

Comment:
No need to expose precision to python bindings, only needed when someone will run routines from the python front end. Can delete this line and file.